lib/image-diff.js

```diff
@@ -54,12 +54,13 @@ ImageDiff.createDiff = function (options, cb) {
   // http://www.imagemagick.org/script/compare.php
   var diffCmd = 'compare';
   var diffArgs = [
-    'compare',
+//    'compare',
     '-verbose',
     // TODO: metric and highlight could become constructor options
     '-metric', 'RMSE',
+    '-dissimilarity-threshold', '1',
     '-highlight-color', 'RED',
-    '-compose', 'Src',
+    options.shadow[0], options.shadow[1],
```

Comment:
a quick note: the dissimilarity threshold was added because using npm test on even the original tests was failing out due to the images being too dissimilar